From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: celenity <celenity@celenity.dev>
Date: Tue, 28 Jul 2026 01:06:23 +0000
Subject: [PATCH] Gecko - Remote Settings Preview Mode

Allows users to enable Remote Settings Preview Mode if desired.

Signed-off-by: celenity <celenity@celenity.dev>
---
diff --git a/services/settings/Utils.sys.mjs b/services/settings/Utils.sys.mjs
index 76f69ab74e..6183fe4fad 100644
--- a/services/settings/Utils.sys.mjs
+++ b/services/settings/Utils.sys.mjs
@@ -153,7 +153,7 @@ export var Utils = {
   get PREVIEW_MODE() {
     // Release and beta require dev-tools or tests to use preview
     if (
-      AppConstants.RELEASE_OR_BETA &&
+      false &&
       !lazy.isRunningTests &&
       Services.env.get("MOZ_REMOTE_SETTINGS_DEVTOOLS") !== "1"
     ) {
-- 
